[skip vbump] closes #276 release 0.6.0 (#277)
pharmaverse/admiralvaccine: NEWS.md

# admiralvaccine 0.6.0

## Documentation

- In the function documentation references to vignettes were updated to meet CRAN requirements. (#272)

> - The "Ask AI" widget was added to the bottom right of each page. It enables users to ask questions about {admiralvaccine} and {admiraldev} and receive answers from an LLM. It is trained on the documentation of both packages and provided by kapa.ai. (#271)

Reviewer: ```suggestion
- The "Ask AI" widget was added to the bottom right of each page. It enables users to ask questions about `{admiralophtha}` and the
rest of the `{admiral}` ecosystem and receive answers from an LLM. It is trained on the documentation of all `{admiral}` packages
and provided by [kapa.ai](https://docs.kapa.ai/kapa-for-open-source). (#271)
```

Author: why admiralophtha?

Reviewer: sorry of course you're right it should be vaccine. force of habit!

> - A link to the {admiral} ecosystem page was added to the README sidebar and main text. (#270)

Reviewer: ```suggestion
- A link to the `{admiral}` ecosystem page was added to the README sidebar and main text. (#270)
```
